frontend/src/helpers/wrap-page-element.tsx

```diff
@@ -73,6 +73,29 @@ const isSocial = (props: Props["props"]) => {
   );
 };
 
+const getAlternateLinks = (props: Props["props"]) => {
+  if (typeof window !== "undefined") {
+    /*
+      when we are not SSO,
+      use the location's pathname to generate
+      more specific URLs:
+
+      Example /en/submission/{ID} cannot be generated server side,
+      so we replace it with the current pathname.
+    */
+    const pathname = window.location.pathname
+      .replace("en", "")
+      .replace("it", "");
```

Comment:
wouldn't be better to replace "/en" and "/it"? :)

Reply:
yes

Reply:
I went for /en/ and /it/ so that to remove the entire initial part